### PR TITLE
Separate doc branch

### PR DESCRIPTION
**Pull request by George3d6**
Doc builds are pushed to a separate `gh-pages` branch

Currently https://lightwood.io/ is source from that branch and everything looks peachy :)

### CODE REVIEW

**Review by ZoranPandovski — APPROVED** [review]

File: docs/CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-lightwood.io
```
Comment (paxcema) [inline docs/CNAME:1]: Is it okay for this to be gone?

Comment (paxcema) [reply docs/CNAME:1]: nevermind, it's still in the Makefile :+1:

Comment (ZoranPandovski) [reply docs/CNAME:1]: I think this is required when connecting GitHub pages to a custom domains?

Comment (paxcema) [reply docs/CNAME:1]: Yeah, but it seems it's being generated when doing `make github` in the `doc_build.yml` file, so it _should_ be okay :thinking:

**Pull request merged**